sycl/test-e2e/Annotated_arg_ptr/annotated_ptr.cpp

```diff
@@ -1,3 +1,4 @@
+// REQUIRES: usm_shared_allocations
```

Comment:
Shouldn't this be `aspect-usm_shared_allocations`?

https://github.com/intel/llvm/blob/d3d6e789cd420e88648580c160785bf15200bcc8/sycl/test-e2e/lit.cfg.py#L665-L668

Reply:
Yeah, I noticed that some most tests were using aspect-{ASPECT_NAME} format but it seems that this format works as well. Indeed, if i remove it the tests go back to failing. Perhaps @againull can confirm that both work, that is, using aspect-usm_shared_allocations or just using usm_shared_allocations?

Reply:
I don't think it works, most likely it will never be run. One can achieve the same by `REQUIRES: dont-ever-run-me`.